### PR TITLE
bugfixes for linear binning

### PR DESCRIPTION
**Pull request by Waelthus**
This is needed for linear binning. Else floats will be read as strings... WIP

### CODE REVIEW

**Comment by Waelthus** [comment]
seems to work now, will need to check results

**Comment by Waelthus** [comment]
added a fix reading the new linearly binned deltas. Atm this is just done by converting LAMBDA to log10(λ) on read-in

File: py/picca/data.py [hunk 1]
```diff
@@ -1053,7 +1053,14 @@ def from_fitsio(cls, hdu, pk1d_type=False):
             delta_name = "DELTA"
 
         delta = hdu[delta_name][:].astype(float)
-        log_lambda = hdu['LOGLAM'][:].astype(float)
+
```
Comment (iprafols) [inline py/picca/data.py:1056]: I guess this is necessary to run picca_cf.py right?

**Review by iprafols — APPROVED** [review]

File: py/picca/data.py (hunk 1 above)
Comment (Waelthus) [reply py/picca/data.py:1056]: yes, else picca won't be able to read anything for downstream analysis. What I do not know if inside e.g. picca_cf we implicitely assume equally sampled log_lambda. So this is just a placeholder change until we have modified routines to properly deal with linear binning

**Pull request merged**